Openweathermap (#1204)
n8n-io/n8n-docs: docs/integrations/builtin/app-nodes/n8n-nodes-base.openweathermap.md

# OpenWeatherMap

> The OpenWeatherMap node allows you to automate work in OpenWeatherMap, and integrate OpenWeatherMap with other applications. n8n has built-in support for a wide range of OpenWeatherMap features, including returning weather data. 

Reviewer: ```suggestion
The OpenWeatherMap node allows you to automate work in OpenWeatherMap, and integrate OpenWeatherMap with other applications. n8n supports retrieving current and upcoming weather data with OpenWeatherMap.
```
